action.yml

```
@@ -64,21 +50,21 @@ runs:
           RELEASES=$(curl -s https://api.github.com/repos/julelang/jule/releases)
           LATEST_RELEASE=$(echo "$RELEASES" | jq -r '.[] | select(.draft == false) | .tarball_url' | head -n 1)
 
-          wget $LATEST_RELEASE -O julec.tar.gz
+          curl -o ./julec.tar.gz $LATEST_RELEASE
```

No need to use `./` (same in other parts of the code).
```suggestion
          curl -o julec.tar.gz $LATEST_RELEASE
```